test/jit/test_class_type.py

```diff
@@ -1167,7 +1167,7 @@ def free_function(x: int) -> int:
 
         @torch.jit.script
         class Properties(object):
-            __ignored_properties__ = ["unsupported"]
+            __jit_unused_properties__ = ["unsupported"]
```

Comment:
Ideally we don't need to add another class attribute API and `__jit_ignored_attributes__` would still work

Reply:
The issue is that properties are not copied to the `RecursiveScriptModule` wrapper, so they can only be used in script mode (even when they are legal; this is due to limitations with [dynamically adding properties](https://stackoverflow.com/a/1355444) but it's possible that it can be worked around). So if someone put the name of a property into `__jit_ignored_attributes__` and tried to call it, they might be confused when it doesn't work.